pennylane_lightning/src/gates/cpu_kernels/avx_common/ApplyCY.hpp

```diff
@@ -0,0 +1,298 @@
+// Copyright 2022 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * @file
+ * Defines CY gate
+ */
+#pragma once
+#include "AVXConceptType.hpp"
+#include "AVXUtil.hpp"
+#include "BitUtil.hpp"
+#include "Blender.hpp"
+#include "Permutation.hpp"
+
+#include "ConstantUtil.hpp"
+#include "Util.hpp"
+
+#include <immintrin.h>
```

Comment:
Do we need this header if the intrinsic calls are defined in another templated header? 

If we do, all good. If not, we should remove it wherever we do not explicitly need it defined.

Reply:
Yeap removed this header from all `Apply*.hpp` files.